modVM/src/org/aion/vm/BulkExecutor.java

```diff
@@ -449,4 +450,22 @@ private boolean isContractAddress(Address address) {
             return (code != null) && (code.length > 0);
         }
     }
+
+    private byte getVmType(Address destination) {
+        if (ContractFactory.isPrecompiledContract(destination)) {
+            // skip the call to disk
+            return TransactionTypes.FVM_CREATE_CODE;
+        } else {
+            byte storedVmType = repositoryChild.getVMUsed(destination);
+
+            // DEFAULT is returned when there was no contract information stored
+            if (storedVmType == TransactionTypes.DEFAULT) {
```

Comment:
Again, is this supposed to be an error?

Reply:
This value will be returned for contracts that have not been stored, resulting from databases that have been migrated to the new build without re-importing all blocks. In that case, we check for the vm type stored in the contract details object.